packages/ckeditor5-dev-release-tools/tests/tasks/commitandtag.js

```diff
@@ -123,4 +127,19 @@ describe( 'commitAndTag()', () => {
 
 		expect( stubs.git.addTag ).toHaveBeenCalledExactlyOnceWith( 'v1.0.0' );
 	} );
+
+	it( 'should run in dry run mode without creating a commit and a tag', async () => {
```

Comment:
Could you add one more test to verify if the specified `options.preCommitCommand` value is passed to `shExec()`?

Reply:
Also, I do not see anything regarding to `listrTask.output` in tests.

Reply:
Adressed in [d47c4a6](https://github.com/ckeditor/ckeditor5-dev/pull/1077/commits/d47c4a68ce779b98208229a04d93400ea3c3dba3).